### DISCUSSION TITLE
Why isn't it possible to return a psql type from a custom sql function in hasura?

### DISCUSSION THREAD

**Discussion by reinoldus** [hasura/graphql-engine#6472] [Question]
As stated in the tile: Why isn't it possible to return a psql type from a custom sql function in hasura? https://www.postgresql.org/docs/9.2/sql-createtype.html
 
The only alternative I see around this is to create an empty table to return the data directly through hasura or of course a custom action.

Am I missing something?

**Comment by michael-land** [comment]
+1 for return a psql type feature
